distribution/scripting.md

```diff
@@ -129,6 +132,8 @@ Game actions allow you to define new actions (with a new permission slot) that p
 
 This sequence of actions ensures that every player execute the action exactly in the same way on the exact same game tick. It also allows the server to validate that the action is allowed and does not fail due to permission or another player executing a conflicting action just before it. This is why there is a noticeable delay when constructing in multiplayer games, as the client has to wait for the server to acknowledge the action and reply with the tick number to execute it on, since we do not yet have any rollback support.
 
+Besides built-in game actions, remote scripts can also register their own game actions (“custom game actions”). These custom actions allow custom game state mutations to be executed in sync for all players on the server. Beware that the plugin will also be responsible for proper permission checks, as custom game actions have no permission checks by default.
```

Comment:
Technically, any plugin can define and use custom actions, not just remote plugins.

Reply:
Technically yes, but practically there's not much benefit. In multiplayer clients who don't have the local plugin won't execute the custom action.